package.json

```diff
@@ -103,6 +103,7 @@
     "@elastic/apm-rum-react": "^2.0.2",
     "@elastic/charts": "63.1.0",
     "@elastic/datemath": "5.0.3",
+    "@elastic/ecs": "8.12.0-dev.1",
```

Comment:
An unexpected side effect of this versioning is it's showing up in persisted values on the DQ dashboard:

<img width="1525" alt="ecsVersion" src="https://github.com/elastic/kibana/assets/4459398/9bf2ac71-5d3d-445e-9742-2d3060d303f9">

Reply:
it's the actual ecs version :| would it make sense to drop the suffix?

Reply:
The `-dev` version also appears in the UI (and generated markdown):

<img width="1243" alt="version-with-dev" src="https://github.com/elastic/kibana/assets/4459398/d315391d-3d53-4b57-9c87-333892174459">

Reply:
Consider using the default `^` syntax, e.g. `"@elastic/ecs": "^8.12.0",`, because any minor and patch versions to ECS should be additions of fields (or fixes) that don't break compatibility

Reply:
done!